courses/free-courses-ta.md

```diff
@@ -0,0 +1,62 @@
+### Index
+
+* [Android](#android)
+* [Bash and Shell](#bash-and-shell)
+* [C](#c)
+* [C++](#cpp)
+* [HTML and CSS](#html-and-css)
+* [Java](#java)
+* [JavaScript](#javascript)
+* [PHP](#php)
+* [Python](#python)
+
+
+### Android
+
+* [Android In Tamil 2019](https://youtube.com/playlist?list=PL4unWLKFsZfcGBja19mrwodNm6AyzZk2B) - Tutor Joe's Stanley
+
+
+### Bash and Shell
+
+* [Bash scripting in Tamil](https://youtube.com/playlist?list=PLgWpUXNR_WCeWiXmsYf5HUe7E4I29zTJr) - Payilagam
+
+
+### C
+
+* [C Tutorial in Tamil](https://youtube.com/playlist?list=PLBQXOA5OR76rq-sU8mNsUmj2Z9kQXz7oD) - Tamil Pro Techniques
+* [Fundamentals of C programming in Tamil](https://youtube.com/playlist?list=PLmjuBlzAWCzxTdRxTtMSj1NFe_GvFlvFw) - Collectiva Knowledge Academy
+
+
+### <a id="cpp"></a>C++
+
+* [C++ Programming in Tamil](https://youtube.com/playlist?list=PLWbtDrDnmTHBPD-Dt5BJi7iP11x6UvgU0) - CS in Tamil
+
+
+### HTML and CSS
+
+* [CSS in Tamil](https://youtube.com/playlist?list=PL73Obo20O_7gGv4cLEOoqTF8_m8rPKyQh) - CyberDude Networks Pvt. Ltd.
+* [HTML Course in Tamil](https://youtube.com/playlist?list=PL73Obo20O_7gcXt0cfQA14jey8zavtKAq) - CyberDude Networks Pvt. Ltd.
+* [Learn HTML in Tamil \| HTML Tutorials in Tamil \| Beginner to Advanced](https://youtube.com/playlist?list=PLpYn3LR7eQI2trAr1z1lmvYGAQfsbllY5) - Tamil Developer
+
+
+### Java
+
+* [Learn Java in Tamil](https://youtube.com/playlist?list=PLYM2_EX_xVvVXm005Gt5unmqW6GGMjHxa) - Logic First Tamil
+
+
+### JavaScript
+
+* [Javascript Full Course in Tamil - in Depth Javascript Tutorials in Tamil](https://youtube.com/playlist?list=PLpYn3LR7eQI3hjh129Bkqkw7onut28hTK) - Tamil Developer
+* [JavaScript in Tamil](https://youtube.com/playlist?list=PL73Obo20O_7ihsIM5K-hHYPrcqkkdQcLa) - CyberDude Networks Pvt. Ltd.
+
+
+### PHP
+
+* [PHP for beginners In Tamil](https://youtube.com/playlist?list=PL4unWLKFsZfcq_D-sEy0pR4Sl_yipy6Jt) - Tutor Joe's Stanley
+* [PHP Tutorial In Tamil](https://youtube.com/playlist?list=PL4unWLKFsZfdrMitLmm8N-idlYQkSCvT9) - Tutor Joe's Stanley
+
+
+### Python
+
+* [Python in Tamil for Beginners](https://youtube.com/playlist?list=PLA2UBjeRwle3OLO3qmXTbmCvuTlqhHRVb) - GURUKULA
+* [PYTHON TUTORIAL IN TAMIL](https://youtube.com/playlist?list=PLIFRUdRwOM0_hcLruKbsHWnU5P2uLBgsp) - SANTRA TECHSPOT
```

Comment:
From CONTRIBUTING.md 

> - No ALLCAPS titles. Usually title case is appropriate, but when doubt use the capitalization from the source

Reply:
What do you mean sir?

Reply:
Please read CONTRIBUTING.md guidelines!

Reply:
Ok

Reply:
> Please read CONTRIBUTING.md guidelines!

Sir how to solve this

Reply:
Source is in ALLCAPS but... here the title-case patch...

```suggestion
* [Python Tutorial in Tamil](https://youtube.com/playlist?list=PLIFRUdRwOM0_hcLruKbsHWnU5P2uLBgsp) - SANTRA TECHSPOT
```

Reply:
Resolved by a813fdf71ee1a0f09978e1e781d0cdb6ac2da885